.github/workflows/README.md

```diff
@@ -0,0 +1,236 @@
+![](https://github.com/nektos/act/wiki/img/logo-150.png)
```

Comment:
this again seems to be added by the act tool which we are not using. Please remove it.